### PR TITLE
docs: MAINTAINERS file and LGTM configuration

### PR DESCRIPTION
**Pull request by jirikuncar**
Signed-off-by: Jiri Kuncar <jiri.kuncar@cern.ch>

### CODE REVIEW

**Comment by jirikuncar** [comment]
LGTM

File: MAINTAINERS [hunk 1]
```diff
@@ -0,0 +1 @@
+Jiri Kuncar <jiri.kuncar@cern.ch> (@jirikuncar)
```
Comment (tiborsimko) [inline MAINTAINERS]: Please add also:

```
Lars Holm Nielsen <lars.holm.nielsen@cern.ch> (@lnielsen)
```

to seed up the system faster.

Comment (jirikuncar) [inline MAINTAINERS]: done

**Comment by jirikuncar** [comment]
LGTM

**Comment by tiborsimko** [comment]
LGTM

**Pull request merged**